config/api.yml

```diff
@@ -617,6 +617,37 @@
       :post:
       - :name: create
         :identifier: picture_new
+  :actions:
+      :description: Actions
+      :identifier: action
+      :options:
+      - :collection
+      :verbs: *gpd
+      :klass: MiqAction
+      :collection_actions:
+        :get:
+        - :name: read
+          :identifier: action_show_list
+        :post:
+        - :name: query
+          :identifier: action_show_list
+        - :name: create
+          :identifier: action_new
+        - :name: edit
+          :identifier: action_edit
+        - :name: delete
+          :identifier: action_delete
+      :resource_actions:
+        :get:
+        - :name: read
+          :identifier: action_show
+        :post:
+        - :name: edit
+          :identifier: action_edit
+        - :name: delete
+          :identifier: action_delete
+        - :name: create
```

Comment:
@alongoldboim when you get a chance, could you update the api.yml to remove create from resource_actions ?  Thanks.